web/modules/custom/mymodule/src/Controller/FirstController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file 
+ * Generate markup to be displayed. Functionality in this controller is
+ * wired to Drupal in mymodule.routing.yml.
+ */
+
+namespace Drupal\mymodule\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+
+class FirstController extends ControllerBase {
+
+  public function simpleContent() {
+    return [
+      '#type' => 'markup',
+      '#markup' =>('hello world'),
+    ];
+  }
+
+  public function greetings($name) {
+    return[
```

Comment:
Write it in either of the following 2 ways only:
```
    return[
      '#type' => 'markup',
      '#markup' =>t('HELLO @name !', ['@name' => $name]),
    ];
```
OR 
```
    return[
      '#type' => 'markup',
      '#markup' =>t('HELLO @name !', [
        '@name' => $name
       ]),
    ];
```

Reply:
Rectified. Used the second method.